### PR TITLE
Portfolio Desktop Version

### PR DESCRIPTION
**Pull request by Loltolo-Lesapiti**
This project used media queries to ensure that the portfolio website displays correctly on desktop screens.

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -0,0 +1,547 @@
...
+
+.home {
+  background-image: url("images/mobile.png");
+  background-repeat: no-repeat;
+  background-position: top;
+  background-size: 375px 588px;
+  border-bottom-left-radius: 20px;
+  margin-bottom: 30px;
+  background-color: #fff;
+}
```
Comment (cynthiainga) [inline styles.css:79]: - [ ] Please your design must stick to the Figma design as much as possible, you need to add the `border-radius` to the left of your `home` section :upside_down_face: :upside_down_face: 

Your design

Screenshot from 2022-07-08 13-04-07

Figma design

Screenshot from 2022-07-08 13-03-23

- [ ] Also add more space between your nav-menu :grin: :grin:
- [ ] I noticed that your nav-menu color does not match the Figma design, your design must stick to the design as much as possible (e.g., font, colors, images, text, margins), please kindly fix it.

File: styles.css [hunk 2]
```diff
@@ -0,0 +1,547 @@
...
+  margin-left: 0;
+}
+
+.about {
+  display: grid;
+  gap: 80px;
+  border-top-right-radius: 30px;
+  background: #fff;
+  padding: 20px;
+}
```
Comment (cynthiainga) [inline styles.css:209]: - [ ] Same remarque about `border-radius`, `colors`, `font`..., please kindly make sure your border-radius is implemented correctly :upside_down_face: :upside_down_face:. Please your design must stick to the design as much as possible.

Your design

Screenshot from 2022-07-08 13-05-05

Figma design
Screenshot from 2022-07-08 13-05-24

**Review by cynthiainga — CHANGES_REQUESTED** [review]
Hi @Loltolo-Lesapiti, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights

- [ ] All linters are passing :heavy_check_mark: 
- [ ] Cood commit history :heavy_check_mark: 
- [ ] Meaningful pull request name :heavy_check_mark: 

## Required Changes ♻️

- [ ] Please kindly check the comments under the review for more details :point_down: :point_down:

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- [ ] Please put in mind that since it's a solo project, It should be worked on individually :upside_down_face: :upside_down_face:. I see some commits of your partners while the activity is solo. Pay attention to that.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Loltolo-Lesapiti** [comment]
The commits from my coding partners were from the previous day's assignment. I created a new branch for this assignment.

File: styles.css [hunk 3]
```diff
@@ -0,0 +1,550 @@
...
+  /* -------Home section--------- */
+  .home {
+    background-color: #ffff;
+    border-bottom-left-radius: 80px;
+    background-image: url("images/headerbg.png");
+    background-repeat: no-repeat;
+    background-size: contain;
+    background-position-y: 77px;
+    padding-bottom: 2%;
+  }
```
Comment (vechicin) [inline styles.css]: While you did address the previous reviewer's requirement, you didn't do so properly. Please make sure the background image is placed correctly. Refer to the Figma template to do so.

File: styles.css [hunk 4]
```diff
@@ -0,0 +1,550 @@
...
+
+  .menu_desktop {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    padding: 0 5%;
+    background: #fff;
+    position: fixed;
+    width: 100%;
+  }
```
Comment (vechicin) [inline styles.css]: The margin between your navbar and the section below is not correct. Please fix it.

File: styles.css [hunk 5]
```diff
@@ -0,0 +1,550 @@
...
+
+  /* -----------About section------------ */
+  .about {
+    display: flex;
+    flex-direction: row;
+    gap: 25px;
+    padding: 50px;
+    border-top-right-radius: 80px;
+  }
```
Comment (vechicin) [inline styles.css]: The margin with the section above is not right. Please fix it.

File: styles.css [hunk 6]
```diff
@@ -0,0 +1,550 @@
...
+
+  .submit {
+    align-self: center;
+  }
+
+  .contact_section {
+    background-image: url(images/contact_bg.png);
+    background-repeat: no-repeat;
+    background-size: contain;
+  }
```
Comment (vechicin) [inline styles.css:541]: You have a vertical overflow issue in this section. Please fix it.

Screen Shot 2022-07-08 at 8 26 38

**Review by vechicin — CHANGES_REQUESTED** [review]
# Hi @Loltolo-Lesapiti ✋, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet. 🚨 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions 📂 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 7]
```diff
@@ -0,0 +1,208 @@
...
+                </div> 
+                <p class="card-info">
+                    A daily selection of privately personalized reads; no accounts or sign-ups required.
+                </p>
+                <ul class="list second">
+                <li>html</li>
+                <li>css</li>
+                <li>javascript</li>
+                </ul>
+                <button class="see-more">See Project</button>
```
Comment (Nemwel-Boniface) [inline index.html:82]: - [ ] It appears that the new changes that were introduced broke the design for your buttons. Right now your buttons have an overflow and the text does not fit within the button. Kindly fix this by reducing the size of the button text and the project will be ready for approval.

**This is what your buttons look like:**
image

**This is how the buttons should look like:**
image

File: index.html [hunk 8]
```diff
@@ -0,0 +1,208 @@
...
+                <p class="card-info">
+                    Experimental content creation feature that allows users to add to an existing story over the course of a day without spamming their friends.
+                </p>
+                <ul class="list second">
+                <li>html</li>
+                <li class="ruby">Ruby on rails</li>
+                <li>css</li>
+                <li>javascript</li>
+                </ul>
+                <button class="see-more">See Project</button>
```
Comment (Nemwel-Boniface) [inline index.html:106]: - [ ] It appears that the new changes that were introduced broke the design for your buttons. Right now your buttons have an overflow and the text does not fit within the button. Kindly fix this by reducing the size of the button text and the project will be ready for approval.

**This is what your buttons look like:**
image

**This is how the buttons should look like:**
image

File: index.html [hunk 9]
```diff
@@ -0,0 +1,208 @@
...
+                <p class="card-info">
+                    Exploring the future of media in Facebook's first Virtual Reality app; a place to discover and enjoy 360 photos and videos on Gear VR.
+                </p>
+                <ul class="list second">
+                <li>html</li>
+                <li class="ruby">Ruby on rails</li>
+                <li>css</li>
+                <li>javascript</li>
+                </ul>
+                <button class="see-more">See Project</button>
```
Comment (Nemwel-Boniface) [inline index.html:130]: - [ ] It appears that the new changes that were introduced broke the design for your buttons. Right now your buttons have an overflow and the text does not fit within the button. Kindly fix this by reducing the size of the button text and the project will be ready for approval.

**This is what your buttons look like:**
image

**This is how the buttons should look like:**
image

File: index.html [hunk 10]
```diff
@@ -0,0 +1,208 @@
...
+                </div> 
+                <p class="card-info">
+                    A daily selection of privately personalized reads; no accounts or sign-ups required.
+                </p>
+                <ul class="list second">
+                <li>html</li>
+                <li>css</li>
+                <li>javascript</li>
+                </ul>
+                <button class="see-more">See Project</button>
```
Comment (Nemwel-Boniface) [inline index.html:153]: - [ ] It appears that the new changes that were introduced broke the design for your buttons. Right now your buttons have an overflow and the text does not fit within the button. Kindly fix this by reducing the size of the button text and the project will be ready for approval.

**This is what your buttons look like:**
image

**This is how the buttons should look like:**
image

**Review by Nemwel-Boniface — CHANGES_REQUESTED** [review]
## Hello @Loltolo-Lesapiti     :wave: , 
I'm Nemwel your reviewer for this PR :man_beard: 

Good job so far in setting your portfolio desktop version. I commend you because I really like the design! :clap: 
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights

Great job setting up the correct linters ✔️ 
Professional documentation ✔️ 
Correct workflow ✔️ 
Meaningful commit messages ✔️


## ♻️ Required Changes ♻️

_Check the comments under the review._

## [Optional] suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding! :partying_face: 👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me **@Nemwel-Boniface** in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 10 above)
Comment (Loltolo-Lesapiti) [reply index.html:153]: Hello, I cannot identify the overflow in my buttons on my local machine, can you send the breakpoints please.

**Review by Lucas-Masaba — APPROVED** [review]
# Hello @Loltolo-Lesapiti  👋 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

almost-there

## Highlights 🍾 
- Linter checks passing ✔️ 
- Done most changes requested by the previous reviewer 👍 
- README is looking professional 📜 

## Optional suggestions 💭 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏 💻 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**